### PR TITLE
Code review1 mastroja

### PR DESCRIPTION
**Pull request by mastroja**

### CODE REVIEW

**Comment by mastroja** [comment]
Overall Project is good, and it looks like you have made a good start on the styling of the page.  Most of the changes I added were not based on issues within the app but changes I thought were more fitting to the style of coding our professor has shown thus far.

-added a note in the README.md file, just needs a little update so future reviewers can better understand.

-There is a Activity.java file which I moved into a DTO class, and edited some of the Service files to help implement that activityDTO.

-Added some mapping to the controller, which probably wasn't needed, but again I was just adding things to better match examples we have been shown in class. Which to be honest it may be best to stick as close to those examples as possible just for final grading purposes.

1.) Analysis of the program
There we not many changes that i needed to add, the code looks good and i like the styling so far.  Going forward maybe just watch some of the naming conventions (i.e weeklyScheduleServiceImpl.java) and just make it something a little cleaner/easier to understand.  Good work and thanks again for letting me review you project on such short notice!

2.)Was the program available in UC Github on time?
Yes

3.)Is the program documented/commented well enough for you to understand?
some work needed on the README, otherwise, it is good

4.)Does the program compile?
Yes

**Comment by mastroja** [comment]
three specific technical concepts that you learned from this code review

I dont feel like i learned anything new from this review, but rather saw more examples of what we have already learned. i.e. service stubs and dto working together.
There was some javascript in the home.html file, and I always like seeing some javascript

File: EisenBoard/src/main/java/com/EisenBoard/DTO/ActivityDTO.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package com.EisenBoard.DTO;
```
Comment (discospiff) [inline EisenBoard/src/main/java/com/EisenBoard/DTO/ActivityDTO.java]: Package names should be lowercase in Java, even if it's an acronym.

File: EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleService.java [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+
+import org.springframework.stereotype.Service;
+
+import com.EisenBoard.DTO.ActivityDTO;
+
+import main.java.com.EisenBoard.WebAppDev.model.Activity;
+
+
+@Service
+public interface weeklyScheduleService {
```
Comment (discospiff) [inline EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleService.java]: Class names should be UpperCamelCase in Java.... as long as you're making changes to this class, might as well fix the naming conventions.

File: EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleServiceImpl.java [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+import com.EisenBoard.DTO.ActivityDTO;
+
+import main.java.com.EisenBoard.WebAppDev.model.Activity;
+
+public class weeklyScheduleServiceImpl implements weeklyScheduleService {
+
+	@Override
+	public ActivityDTO fetchById(int id) {
+		// TODO Auto-generated method stub
+		return null;
```
Comment (discospiff) [inline EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleServiceImpl.java]: This is a no-no, as the method is not doing what the name indicates.  Also, it's an incomplete/unused feature.  This is technical debt.

File: EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleServiceImpl.java [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
+	}
+
+	@Override
+	public void save(ActivityDTO activityDTO) {
+		// TODO Auto-generated method stub
+
```
Comment (discospiff) [inline EisenBoard/src/main/java/com/EisenBoard/WebAppDev/service/weeklyScheduleServiceImpl.java]: Ditto comment above.  An empty/unused method is clutter, which is technical debt.

File: README.md [hunk 5]
```diff
@@ -6,3 +6,6 @@ Scrum Roles
 Project Manager - Jerry
 Front-end -
 Back-end -
+
+
+*add more details about what the app is meant to do, as well as some instructions on how to run your app. (i know most of the apps will be ran in the same way, but for example your index page is called home, while the one for my project is called start so little differences that may cause confusion)
```
Comment (discospiff) [inline README.md]: The readme.md should be updated, but this falls under the category of, "don't add a comment, telling the group what to do.  Demonstrate it for them, in code."  If you're going to update the readme, update it with useful information.

**Comment by discospiff** [comment]
I don't recommend merging these changes.

While I see a bit of effort in this code review, the changes introduce more technical debt than they resolve.

**Pull request closed without merging**